lib/views/post/widgets/post_buttons.dart

```diff
@@ -1,3 +1,4 @@
+import 'package:flare_flutter/flare_actor.dart';
```

Comment:
Split the widget. It does more then the name suggests. Introduce a PostInfoBar Widget which contains the user and benis score and also the PostButton widget, which shall only handle the buttons

Reply:
#14 resolves this